test/oxo.js

```diff
@@ -0,0 +1,274 @@
+const Reverter = require('./helpers/reverter');
+const Asserts = require('./helpers/asserts');
+const OXO = artifacts.require('./OXO.sol');
+
+contract('OXO', function(accounts) {
+  const reverter = new Reverter(web3);
+  afterEach('revert', reverter.revert);
+
+  const asserts = Asserts(assert);
+  const OWNER = accounts[0];
+  let oxo;
+
+  before('setup', () => {
+    return OXO.deployed()
+    .then(instance => oxo = instance)
+    .then(reverter.snapshot);
+  });
+
+  it('should not allow to mark before bids are made', async () => {
+    const player1 = accounts[3];
+    const player2 = accounts[4];
+
```

Comment:
you use player 1 and player2 variables throw the all test. Why son't you take them out in the global variables?

Reply:
will you have some more tests?
Like check if it's possible to play next game

Reply:
@Purka done ;)